docs/plugins/salesforce.rst

```diff
@@ -3,4 +3,6 @@
 Salesforce
 ##########
 
-.. vale on
\ No newline at end of file
+.. vale on
+
+Visit [Knowledge base](https://kb.mautic.org/article/how-to-integrate-mautic-with-salesforce.html) for detains on how to configure the Samesforce integration.
```

Comment:
```suggestion
Visit the [Knowledgebase](https://kb.mautic.org/article/how-to-integrate-mautic-with-salesforce.html) for details on how to configure the Salesforce integration.
```

Reply:
Could you please use the command 'make link' at the console to create the link file, and replace this with the xref macro it provides you with? You'll need to commit the link file, also. Thanks!

Reply:
@RCheesley do you have docs or an example about the `make link` command? I get this:

```
make link
make: *** No rule to make target `link'.  Stop.
```

I was looking into the `--help` in CLI but I have no idea what option to use. I also searched this repo for some docs about the `make link` but didn't find anything.

Reply:
You need to be in /docs to run the command.

Reply:
That worked!

Reply:
Great! FWIW the docs are here: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages